src/autora/state/wrapper.py

```diff
@@ -58,9 +109,42 @@ def experiment_runner(conditions: pd.DataFrame, **kwargs):
     return experiment_runner
 
 
-def experiment_runner_from_x_to_xy_function(f: Callable[[X], XY]) -> Executor:
+def state_fn_from_x_to_xy_fn_df(f: Callable[[X], XY]) -> Executor:
     """Wrapper for experiment_runner of the form $f(x) \rarrow (x,y)$, where `f`
-    returns both $x$ and $y$ values in a complete dataframe."""
+    returns both $x$ and $y$ values in a complete dataframe.
+
+    Examples:
+        The conditions are some x-values in a StandardState object:
+        >>> from autora.state.bundled import StandardState
+        >>> s = StandardState(conditions=pd.DataFrame({"x": [1, 2, 3]}))
+
+        The function can be defined on a DataFrame, allowing the explicit inclusion of
+        metadata like column names.
+        >>> def x_to_xy_fn_df(c: pd.DataFrame) -> pd.Series:
```

Comment:
why is the final bit `_df` included in the name of the function being wrapped in this example but no other ones (as far as i can tell)? ie, it's not part of functions defined in lines 77, 89, 135.

Reply:
oh right, good point. That's not necessary. Fix coming.